using-mobile-extensions/adobe-target/README.md

```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
+While merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary while the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by those from global parameters. In the case or `TargetOrder` or `TargetProduct` objects, the object passed in global parameters shall replace the corresponding object in each of `TargetRequest` or `TargetPrefetch` objects.
 
-{% tabs %}
-{% tab title="Android" %}
-#### **Syntax**
+## Target Sessions
 
-```java
-public static void setPreviewRestartDeepLink(final Uri deepLink);
-```
+With Target release v2.1.4 iOS and  v1.1.3 Android, Target extension now supports persistent sessions. When a Target request is received, a new session ID is generated if one doesn't already exist and is sent in the request. This session ID along with the Edge Host returned from the Target are stored in the persistent storage for the configured `target.sessionTimeout`. If not configured, the default value for the session timeout is 30 minutes. These variables are reset and removed from persistent storage if no Target request is received during the configured `target.sessionTimeout` or if  [resetExperience API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#reset-user-experience) is called.
 
-#### **Example**
-
-```java
-Target.setPreviewRestartDeepLink("myApp://HomePage");
-```
-{% endtab %}
+## Visual preview
 
-{% tab title="iOS" %}
-#### Syntax
+Visual preview mode allows you to easily perform end-to-end QA for Target activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html).
 
-```text
-+ (void) setPreviewRestartDeepLink: (nonnull NSURL*) deepLink;
-```
+You can also set an application deep link that can be triggered when selections are made in the preview mode by using the [setPreviewRestartDeeplink API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#set-preview-restart-deep-link)
 
-#### **Objective-C Example**
+The `collectLaunchInfo` API is used to enter the visual preview mode. After the visual preview mode is enabled, a red floating button is displayed on the app screen. This button can be pressed to enter the visual preview mode.
 
-```text
-[ACPTarget setPreviewRestartDeepLink:@"myApp://HomePage"];
-```
+{% tabs %}
+{% tab title="Android" %}
 
-#### **Swift Example**
+On Android, when the application is launched as a result of a deeplink, `collectLaunchInfo` API will be invoked internally with the target Activity and deeplink information will be extracted from the Intent extras.
```

Comment:
- Please reword as follows:
"In Android, when the application is launched as a result of a deep link, the `collectLaunchInfo` API is internally invoked, and the target Activity and deep link information is extracted from the Intent extras."

Some questions:
1. Is "target" referring to the solution? Should it be "Target"?
2. Do "Activity" and "Intent" need to be capitalized?

Reply:
I've capitalized `Target` and left Activity and Intent capitalized as they are java classes.